File: gerenciador de tarefas - final/models/Lista.php
<?php 

class Lista{
    private $titulo;
    private $itens;

    public function __construct($titulo){
        $this->titulo = $titulo;
        $this->itens = array();
    }

	public function getTitulo() {
		return $this->titulo;
	}
	
	public function setTitulo($titulo): self {
		$this->titulo = $titulo;
		return $this;
	}

	public function getItens() {
		return $this->itens;
	}
	
    public function adicionarItem($item){
		$this->itens[] = $item;
    }
    
    public function excluirItem($indice){
        if (isset($this->itens[$indice])) {
            unset($this->itens[$indice]);
            $this->itens = array_values($this->itens);
        }
    }
}
?>

File: java desktop/src/main/java/com/taniele/java_desktop/model/Lista.java
package com.taniele.java_desktop.model;

import java.util.ArrayList;
import java.util.Collections;
import java.util.List;

public class Lista {
    private String titulo;
    private List<Item> itens;

    public Lista(String titulo) {
        this.titulo = titulo;
        this.itens = new ArrayList<>();
    }

    public String getTitulo() {
        return titulo;
    }

    public void setTitulo(String titulo) {
        this.titulo = titulo;
    }

    public List<Item> getItens() {
        return itens;
    }

    public void adicionarItem(Item item){
        itens.add(item);
        Collections.sort(itens);
    }

    public void excluirItem(int indice){
        if (indice >= 0 && indice < itens.size()) {
            itens.remove(indice);
        }
    }
}
